src/pretalx/orga/forms/cfp.py

```diff
@@ -153,22 +156,45 @@ def __init__(self, *args, event=None, **kwargs):
         ):
             self.fields["is_public"].disabled = True
 
+    def clean(self):
+        deadline = self.cleaned_data["deadline"]
+        question_required = self.cleaned_data["question_required"]
+        if (deadline == "" or deadline is None) and (
+            question_required == "require after" or question_required == "freeze after"
+        ):
+            raise forms.ValidationError(
+                "If you select 'freeze after deadline' or 'require after deadline' choice you "
+                + "should select the date and time deadline."
+            )
+        if (deadline != "" and deadline is not None) and (
+            question_required == "none" or question_required == "require"
+        ):
+            raise forms.ValidationError(
```

Comment:
Please make the errors translated strings using `_()`.

Reply:
🗺I fixed that for the ValidationError message in the file src/pretalx/orga/forms/cfp.py.🗺